engine/runner/src/main/resources/META-INF/native-image/org/enso/runner/reflect-config.json

```diff
@@ -1800,458 +1800,6 @@
   "name":"org.enso.base_test_helpers.RangeStream",
   "methods":[{"name":"<init>","parameterTypes":["int", "int"] }]
 },
-{
```

Comment:
Additional NI related fix:

`Persistance` discovers all the available implementations via `META-INF/services/` lookup. That would require reflective access to constructor. However such a discovery is now only performed in _static initializer of the class_ - e.g. during _build time_ of Native image. Hence no reflective operations with `Persistance` subclasses shall be performed during native executable execution. Removing _persistance related debris_ from the reflection configuration file.